content/news/2024-02-06-oscars-project-starting/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: The start of OSCARS - an EOSC project
+date: "2024-02-06"
+tease: "Galaxy Europe being part of the new 'Open Science Clusters' Action for Research & Society' (OSCARS)"
+hide_tease: false
+tags: [oscars]
+subsites: [all-eu, esg]
+main_subsite: eu
+---
+
+[OSCARS logo](/images/logos/OSCARS_logo.png)
+
+The [Albert-Ludwigs-University Freiburg](https://uni-freiburg.de), the Institute for Bioinformatics and the Galaxy Team are happy to announce the participation in the ["**O**pen **S**cience **C**lusters' **A**ction for **R**esearch & **S**ociety" (OSCARS)](https://oscars-project.eu/). With [EuroScienceGateway (ESG)](https://www.eurosciencegateway.org) running in its second year now, OSCARS is an additional project from the [EOSC](https://digital-strategy.ec.europa.eu/en/policies/open-science-cloud) space to be involved in. Similar to ESG, this project offers the opportunity to foster connection to a variety communities beyond life sciences, underlining Galaxy being principally domain-agnostic. OSCARS is financed by the European Commission though the European Research Executive Agency‘s HORIZON-INFRA-2023-EOSC-01-01 funding line, under grant agreement number 101129751.
```

Comment:
Institute of Bioinformatics? I guess its a department in english or something like that. I guess we can drop, maybe?

Reply:
Well, actually I took the intro from the EuroScienceGateway announcement post back in 2022 - considered political reasons here for this sentence. Happy to drop :)

Reply:
Then maybe this should be fixed in both.